tests/api-mongo.test.js

```diff
@@ -259,3 +328,23 @@ describe('MongoDB Words', () => {
     });
   });
 });
+
+describe('API Requests For Home Directory "/"', () => {
+  describe('/undefinedRoute', () => {
+    it('should return response status of 404', (done) => {
+      chai.request(server).get('/undefined').end((_, res) => {
+        expect(res.status).to.equal(404);
+        done();
+      });
+    });
+  });
+
+  it('should return hello world', (done) => {
+    chai.request(server).get('/').end((_, res) => {
```

Comment:
For this test and the other test, you're testing CORS to see if an application that's running on a different host has the ability to request data from the API's host which is http://igboapi.com

Instead of using `chai.request(server)` which tests the local API, you want to make network requests to `igboapi.com` to see if you're able to still grab data from the site.